development/docker-compose.yml

```
@@ -113,6 +113,8 @@ services:
     profiles: [demo, dev]
     build:
       context: ../
+      args:
+        CI_ARG: ${CI}
```

```suggestion
        CI_ARG: ${CI:-0}
```

To avoid warnings when the CI variable is not set.